virttest/bootstrap.py

```diff
@@ -360,6 +365,20 @@ def _forced_or_detected(forced, detected):
         cfg.write("                            - @%s:\n" % release)
         cfg.write("                                variants:\n")
         cfg.write("                                    - @%s:\n" % arch)
+        cfg.write("variants:\n")
+        cfg.write("    - @HostCpuVendor:\n")
+        cfg.write("        variants:\n")
+        cfg.write("            - @%s:\n" % vendor)
+        if family:
+            cfg.write("variants:\n")
+            cfg.write("    - @HostCpuFamily:\n")
+            cfg.write("        variants:\n")
+            cfg.write("            - @%s:\n" % family)
+            if version:
+                cfg.write("                variants:\n")
+                cfg.write("                    - @HostCpuVersion:\n")
+                cfg.write("                        variants:\n")
+                cfg.write("                            - @%s:\n" % version)
```

Comment:
variable name conflict with the one represents host's distribution.

Reply:
@luckyh Tnx!, fixed.